pkg/diff/sql_generator.go

```diff
@@ -51,7 +52,15 @@ var (
 	}
 	migrationHazardSequenceCannotTrackDependencies = MigrationHazard{
 		Type:    MigrationHazardTypeHasUntrackableDependencies,
-		Message: "sequence has no owner, so it can't be tracked. It may be in use by a table or function",
+		Message: "A sequence has no owner, so it can't be tracked. It may be in use by a table or function",
```

Comment:
Use `The` because it's in reference to a specific sequence

Reply:
Hmm, "The" is confusing since it implies that some _other_ sequence may have an owner. I've reworded to `Sequences have no owner, so it can't be tracked. This sequence may be in use by a table or function.`

Reply:
I think it's okay for the `Sequence` wording to be specific since it's located right under the statement of the sequence involved. 

It is possible that all other sequences have an owner